test/RachisTests/AddNodeToClusterTests.cs

```diff
@@ -570,11 +569,24 @@ await cluster.Leader.ServerStore.SendToLeaderAsync(new AddDatabaseCommand(Guid.N
                     using (ctx.OpenReadTransaction())
                     {
                         var record = cluster.Leader.ServerStore.Cluster.ReadDatabase(ctx, db);
-                        Assert.Equal(0, record.DeletionInProgress?.Count ?? 0);
-
+                        var deletionInProgress = record.DeletionInProgress?.Count ?? 0;
+                        var info = "";
                         var topology = record.Topology;
+
+                        if (deletionInProgress > 0 || topology.Members.Count < 2)
+                        {
+                            DatabaseInfo databaseInfo = null;
+                            info += $"deletionInProgress = {deletionInProgress}, status = {record.DeletionInProgress?.FirstOrDefault().Value}. " +
```

Comment:
`record.DeletionInProgress?.FirstOrDefault().Value` will result in NRE if `DeletionInProgress` is empty